src/GitHub.App/Api/ApiClient.cs

```diff
@@ -25,6 +25,7 @@ public partial class ApiClient : IApiClient
     {
         const string scopesHeader = "X-OAuth-Scopes";
         static readonly Logger log = LogManager.GetCurrentClassLogger();
+        static readonly Uri userEndpoint = new Uri("user", UriKind.Relative);
 
         const string ProductName = Info.ApplicationInfo.ApplicationDescription;
```

Comment:
Could you reorder these so that const string and static readonly definitions are grouped together?

Nitpick Nancy on the prowl
![nitpick_nancy](https://cloud.githubusercontent.com/assets/310137/15616769/034ec12e-2403-11e6-9f1c-79e74c0c87e0.jpeg)